cmd/cluster-init/main_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	"github.com/openshift/ci-tools/pkg/testhelper"
+)
+
+func TestInitClusterBuildFarmDir(t *testing.T) {
```

Comment:
Why do we still need this if we are doing the bash-based directory integration test?

Reply:
Discussed in slack. Removing this test.